MekHQ/src/mekhq/gui/adapter/TOEMouseAdapter.java

```diff
@@ -1168,15 +1172,30 @@ private void maybeShowPopup(MouseEvent e) {
                 JMenu i_trn = new JMenu(TOEMouseAdapter.INFANTRY_CARRIERS);
                 JMenu a_trn = new JMenu(TOEMouseAdapter.ASF_CARRIERS);
                 JMenu sc_trn = new JMenu(TOEMouseAdapter.SC_CARRIERS);
+                JMenu singleUnitMenu = new JMenu("");
```

Comment:
Why are you initializing this with a "" instead of just not including text?

Reply:
No reason... <whistles>    Fixed.